### PR TITLE
fix (config-plugins): ManifestServiceAttributes missing foregroundServiceType

### PR DESCRIPTION
**Pull request by micaelomota**
# Why

I'm using expo module-api to create an android foreground service and, Android 10 introduced the `android:foregroundServiceType` attribute within the [`service`](https://developer.android.com/guide/topics/manifest/service-element) element in `AndroidManifest.xml`

 https://developer.android.com/about/versions/14/changes/fgs-types-required

I'm having this typescript error message:

Screenshot 2024-11-21 at 10 35 49 PM

And I noticed this attribute type was missing in the definition:

Screenshot 2024-11-21 at 10 38 04 PM

# How

I just changed the type definition to include `android:foregroundServiceType`

```typescript

type ManifestServiceAttributes = AndroidManifestAttributes & {
    'android:enabled'?: StringBoolean;
    'android:exported'?: StringBoolean;
    'android:foregroundServiceType'?: string;
    'android:permission'?: string;
};
```

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by vonovak — CHANGES_REQUESTED** [review]
Hello and thanks for this PR!

Please make your change in [this file](https://github.com/expo/expo/blob/197cfb33893732c9898feeaa70cabf1bcb6ee6fb/packages/%40expo/config-plugins/src/android/Manifest.ts#L38C6-L38C22)

and then run `yarn build` in the `packages/@expo/config-plugins` directory. Then commit all changes from there. 

Additionally, please add a changelog entry in the `CHANGELOG.md` file in the package.

Thank you!

**Comment by micaelomota** [comment]
Thanks for the review @vonovak  I just pushed the changes

**Review by vonovak — CHANGES_REQUESTED** [review]
Let's please add a changelog entry too.

**Comment by micaelomota** [comment]
@vonovak done! thank you!

File: packages/@expo/config-plugins/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 9.0.10 —  2024-11-24
+
+### 💡 Others
+
+- Add foregroundServiceType property to Android ManifestServiceAttributes by [@micaelomota](https://github.com/micaelomota)) ([#33166](https://github.com/expo/expo/pull/33166) by [@micaelomota](https://github.com/micaelomota))
```
Comment (vonovak) [inline packages/@expo/config-plugins/CHANGELOG.md:17]:
```suggestion
- Add foregroundServiceType property to Android ManifestServiceAttributes by [@micaelomota](https://github.com/micaelomota)) ([#33166](https://github.com/expo/expo/pull/33166)
```

File: packages/@expo/config-plugins/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 9.0.10 —  2024-11-24
+
+### 💡 Others
+
+- Add foregroundServiceType property to Android ManifestServiceAttributes by [@micaelomota](https://github.com/micaelomota)) ([#33166](https://github.com/expo/expo/pull/33166)
```
Comment (vonovak) [inline packages/@expo/config-plugins/CHANGELOG.md:17]:
```suggestion
- Add foregroundServiceType property to Android ManifestServiceAttributes by [@micaelomota](https://github.com/micaelomota)) ([#33166](https://github.com/expo/expo/pull/33166))
```

File: packages/@expo/config-plugins/CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 9.0.10 —  2024-11-24
+
+### 💡 Others
+
+- Add foregroundServiceType property to Android ManifestServiceAttributes by [@micaelomota](https://github.com/micaelomota)) ([#33166](https://github.com/expo/expo/pull/33166))
```
Comment (vonovak) [inline packages/@expo/config-plugins/CHANGELOG.md:17]:
```suggestion
- Add foregroundServiceType property to Android ManifestServiceAttributes ([#33166](https://github.com/expo/expo/pull/33166) by [@micaelomota](https://github.com/micaelomota))
```

**Review by vonovak — APPROVED** [review]

**Pull request merged**